scripts/docker/submarine/submarine-interpreter-cpu-1.0.0/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM zeppelin-tensorflow_1.10-hadoop_3.1.2-cpu:1.0.0
```

Comment:
how is zeppelin-tensorflow_1.10-hadoop_3.1.2-cpu created? it looks like it is built by a script in the PR, how is this name chosen?

Reply:
1. I have carefully considered the issue of the docker name. Otherwise it will be difficult to maintain in the future.
2. Because the name needs to include the name and version number of the main component in the image.
3. **Naming rules**

    + The name `_` indicates the separator, splitting different systems, for example: `tensorflow` and `hadoop`
    + The name`-` indicates the connector, the connection system and the version number, for example: `hadoop-3.1.2`

Reply:
if we are not releasing / publishing this docker image as a part of the Apache Zeppelin release, I think this is ok, otherwise a different naming will need to be used

Reply:
why is cpu at the end? maybe it should be `zeppelin-cpu-tensorflow_1.10-hadoop_3.1.2`?

also any reason for tensorflow 1.10 vs the latest 1.13? https://github.com/tensorflow/tensorflow/releases/tag/v1.13.1

Reply:
About the Docker Image Name, Can you give me some suggestion? Thanks!

Reply:
Yes, tensorflow uses version 1.13.0. I made a mistake and I will correct this error.

Reply:
hmm, it looks to be uncommon to put version number in image name? https://hub.docker.com/u/apache?page=2

other than that I don't have any good/useful suggestion. perhaps the latest you have is fine